expand.py

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/python3
+
+import sys
+import getopt
+import tempfile
+import subprocess
+
+usage = '''Usage:expand.py [options] <output modules>
+Output Modules:
+    convolution
+    dsu
+    fenwicktree
+    lazysegtree
+    math
+    maxflow
+    mincostflow
+    modint
+    scc
+    segtree
+    string
+    twosat
+
+You can select multiple modules for <output modules>
+    e.g.)expand.py math segtree
+
+Options:
+    -a  --all       import all modules
+    -h  --help      print help
+'''
+output_header = '//https://github.com/rust-lang-ja/ac-library-rs\n'
+opt_list = ['help', 'all']
+output_list_all = ('convolution', 'dsu', 'fenwicktree', 'lazysegtree', 'math',
+                   'maxflow',  'mincostflow', 'modint', 'scc',  'segtree',
+                   'string', 'twosat',
+                   'internal_bit', 'internal_math', 'internal_queue',
+                   'internal_scc', 'internal_type_traits',)
+dependency_list = {'convolution': ('internal_bit', 'modint',),
```

Comment:
- `fenwicktree`: Not used unlike the original ACL.
- `lazysegtree`: In #44, `segtree::Monoid` is re-used.
- `modint`: `internal_type_traits` is not used.
- `segtree`: `internal_type_traits` is used in #37.

```diff
 dependency_list = {'convolution': ('internal_bit', 'modint',),
-                   'fenwicktree': ('internal_type_traits',),
-                   'lazysegtree': ('internal_bit',),
+                   'fenwicktree': (),
+                   'lazysegtree': ('internal_bit', 'segtree'),
                    'math': ('internal_math',),
                    'maxflow': ('internal_type_traits', 'internal_queue',),
                    'mincostflow': ('internal_type_traits',),
-                   'modint': ('internal_math', 'internal_type_traits',),
+                   'modint': ('internal_math',),
                    'scc': ('internal_scc',),
-                   'segtree': ('internal_bit',),
+                   'segtree': ('internal_bit', 'internal_type_traits'),
                    'twosat': ('internal_scc',), }
```